docs/keys.md

```diff
@@ -166,3 +166,18 @@ have my handlers attached:
 ```python
 SHREKLESS_ALT = KC.LALT.clone()
 ```
+
+You can also refer to a key by index:
+* KC['A']
+* KC['NO']
+* KC['LALT']
+
+or the get function which has an optional argument of `default`. `default` is
+be returned if the key is not found (`None` unless otherwise specified):
+* KC.get('A')
+* KC.get('NO', None)
+* KC.get('NOT DEFINED', KC.RALT)
+
+Keys names are case-sensitive.  KC['NO'] being different from KC['no']. It is recommended
+that names are capitalised typically. The exception to this is alphabetical; KC['A'] and
+KC['a'] will both return the same, unshifted, key.
```

Comment:
```suggestion
Key names are case-sensitive. `KC['NO']` is different from `KC['no']`. It is recommended
that names are capitalised typically. The exception to this is alphabetical; `KC['A']` and
`KC['a']` will both return the same, unshifted, key.
```

Reply:
I think "that names are typically capitalised" reads better?

Reply:
also "exception to this is alpha keys: "

Reply:
Have to admit. I'd love to break backwards compatibility and make KC['A'] give back a shifted KC['a']. But that's just me :)

Reply:
You can always open an issue for the community to discuss that.
The problem I see with that is that capitalized letters are the default right now. We need some kind of consistency, especially since ASCII and keycodes are very different things.

Reply:
Oh, I don't care that much! ;)